src/site/content/en/blog/video-basics/index.md

```diff
@@ -0,0 +1,227 @@
+is used, the first 3% of the video is downloaded. Though this shares caveats
+with `'auto'`, downloading just 3% of the video holds a much smaller server/data
+usage cost than the entire video, while still ensuring a portion of the video is
+stored locally for fast video startup.
+
+### preload='none'
+
+This saves the most data, but will lead to slower video startup when play is
+pressed, because as the setting states, zero kilobytes of the video is
+preloaded locally on the device. For videos that are present, but unlikely to
+be played, this is the appropriate setting. This might also be used if the user
+has enabled [Lite mode](https://blog.chromium.org/2019/04/data-saver-is-now-lite-mode.html) in their browser.
+
+## poster
+
+You may want to have a poster image that displays over the video
+window before the video starts playing:
+
+```html
+<video src="myVideo.mp4" poster="/image/myVideoImage.jpg">
+```
+
+<figure class="w-figure">
+  <img src="no-poster.png" alt="A video without a poster shows a black screen before it starts.">
+</figure>
+{% Compare 'worse', 'No poster image' %}
+  A video without a poster shows a black screen before it starts.
+{% endCompare %}
+
+<figure class="w-figure">
+  <img src="poster.png" alt="A video with a poster is much more engaging.">
+</figure>
+{% Compare 'better', 'With a poster image' %}
+  A video with a poster is much more engaging.
+{% endCompare %}
+
+By adding a photo instead of a black box on the page, you make your website more
+engaging and interactive. However, using the `poster` attribute adds an image
+download before the video download begins. For that reason, you might consider
+avoiding adding a poster for videos that autoplay (as the additional download
+will delay the video download).
+
+## Playback controls
+
+Adding a `controls` attribute adds playback controls. Without these, your
+customers cannot start or stop your video. You should add this for videos so
+that users can stop and pause, change the volume, and so on. For background or
+looping videos, you may wish to omit this attribute.
+
+## muted
+
+The `muted` attribute causes playback to begin in a muted state. If no controls
+are supplied, it will remain muted for the entirety of playback. If that is
+intended it might make sense to remove the audio track from the video. This
+further reduces the size of the video file being delivered to the customer.
+
+As with containers and codecs, removing the audio file, also called demuxing, is
+also beyond the scope of this article. You can find instructions in the [Media
+Manipulation Cheat
+Sheet](https://developers.google.com/web/fundamentals/media/manipulating/cheatsheet#demux_split_audio_and_video).
+
+## loop
+
+To deliver a video that loops the content (like an animated GIF), add the `loop`
+attribute. As video files are typically much smaller than animated GIFs, this
+mechanism allows you to [replace your GIFs with video
```

Comment:
We also have a doc on this. https://web.dev/replace-gifs-with-videos/

Feels a little weird to link offsite but I also understand that the author may want to point at their own version. I guess it's ok to leave it or maybe have two links? That feels like it might be odd...